test/views/TaxYearOverviewViewSpec.scala

```diff
@@ -88,11 +91,18 @@ class TaxYearOverviewViewSpec extends TestSupport {
     }
 
     "have a table of income and deductions" which {
-      "has a row for income" in new Setup {
+        "has a row for income when FS IncomeBreakdown is disabled " in new Setup {
```

Comment:
Test should be updated to include a check to determine that a link for income does not appear in this situation.